core/src/rpc.rs

```diff
@@ -383,7 +383,11 @@ impl JsonRpcRequestProcessor {
         let stakes = HashMap::new();
         let stakes = bank.epoch_vote_accounts(epoch).unwrap_or(&stakes);
 
-        Ok(self.blockstore.get_block_time(slot, slot_duration, stakes))
+        Ok(self
+            .blockstore
+            .get_block_time(slot, slot_duration, stakes)
+            .ok()
```

Comment:
@CriesofCarrots this was where we were talking about it might be good to communicate to clients that the slot was deleted

Reply:
Thanks, @carllin, lgtm! I'll expose this with a more robust Rpc response type after it lands